src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/ConfirmOrganizationUserCommand.cs

```diff
@@ -245,14 +250,54 @@ private async Task<IEnumerable<string>> GetUserDeviceIdsAsync(Guid userId)
     }
 
     /// <summary>
-    /// Handles the side effects of confirming an organization user.
-    /// Creates a default collection for the user if the organization
-    /// has the OrganizationDataOwnership policy enabled.
+    /// Creates a default collection for a single user if required by the Organization Data Ownership policy.
+    /// </summary>
+    /// <param name="organizationUser">The organization user who has just been confirmed.</param>
+    /// <param name="defaultUserCollectionName">The encrypted default user collection name.</param>
+    private async Task CreateDefaultCollectionAsync(OrganizationUser organizationUser, string defaultUserCollectionName)
```

Comment:
There is some code duplication here with the `CreateManyDefaultCollectionsAsync` method, but this is the simplest solution for now. I think we should be able to align the code paths once we've built out better bulk sprocs for policy enforcement and default collection creation.